app/assets/stylesheets/application.scss

```diff
@@ -598,28 +585,17 @@ input:focus ~ .highlight {
   background-color: #263238;
 }
 
-#error_explanation {
-  background-color: rgb(255,64,129) !important;
-  min-height: 30px;
-}
-
 .error {
   color: white;
   font-size: 14px;
+  background-color: rgb(255,64,129) !important;
```

Comment:
Color literals like `rgb(255, 64, 129)` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>!important should not be used<br>Commas in function arguments should be followed by a single space